src/components/Books/BooksFilters.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Button from '../../components/Button/Button';
+
+const BooksFilters = props => (
+  <div className="filters">
+    <Button
+      onClick={props.onBorrowFilter}
+      type="toggle"
+      className="filter-toggle"
+      text="Books To Borrow"
+    />
+    <Button
+      onClick={props.onHaveFilter}
+      type="toggle"
+      className="filter-toggle"
+      text="Books To Have"
+    />
+  </div>
+);
+
+BooksFilters.propTypes = {
+  onBorrowFilter: PropTypes.func,
+  onHaveFilter: PropTypes.func
+};
+
+export default BooksFilters;
```

Comment:
I think this is over fragmented.

Reply:
I don't agree, since this component has a clear functionality that is separate from the rest.